Update docs for prod environment setting (#3815)
fishtown-analytics/docs.getdbt.com: website/docs/guides/best-practices/environment-setup/3-many-deployment-environments.md

1. You have a **single *development* environment** where dbt users can access the dbt Cloud IDE and make changes to their code. However, you’ll want to update the **[custom branch settings](faqs/Environments/custom-branch-settings)** to ensure that developers create feature branches off of the a non-production branch. For this example, we’ll refer to this as the `qa` branch.
2. You have a **QA deployment environment**, running scheduled jobs from the `qa` branch that deploys your dbt project to a pre-production warehouse location.

> 3. You have a **Production deployment environment,** running scheduled jobs from the `main` branch that deploys your dbt project to your production warehouse location. Be sure to set the environment as production to take advantage of features like dbt Explorer and cross-project refs. 

Reviewer: ```suggestion
3. You have a **Production deployment environment,** running scheduled jobs from the `main` branch that deploys your dbt project to your production warehouse location. 
  * Make sure to set the environment to "Production" so you can take advantage of features like dbt Explorer and cross-project refs. 

```
